templates/master/01-master-kubelet/on-prem/units/kubelet.service.yaml

```diff
@@ -10,7 +10,7 @@ contents: |
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
   ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
-  Environment="KUBELET_LOG_LEVEL=4"
+  Environment="KUBELET_LOG_LEVEL=3"
```

Comment:
Honest question: why not move this its own dropin to `/etc/systemd/system/kubelet.service.d/kubelog.conf` with
```
[Unit]
Environment="KUBELET_LOG_LEVEL=3"
```
That would set the log level in _one_ stop.